### PR TITLE
correct input needs 2 arguments

### PR DESCRIPTION
**Pull request by fchapoton**
#### Release Notes

* combinatorics
  * fixed the input of FpGroup for trivial case

### CODE REVIEW

**Comment by oscargus** [comment]
Clearly this part of the code was not tested earlier. Can you please add a test for it?

**Comment by fchapoton** [comment]
Sorry, but I have no idea of the correct test syntax for sympy. I am a sage person, you know. I will not do anything else here.

**Comment by oscargus** [comment]
OK. Didn't know that. The fix seems reasonable (and I realize you most likely know what you are doing), but with a rather newly assigned merge-access I let someone more experienced merge it.

Thanks!

File: sympy/combinatorics/fp_groups.py [hunk 1]
```diff
@@ -933,7 +933,7 @@ def simplify_presentation(*args, **kwargs):
                                               change_gens=change_gens)
         if gens:
             return FpGroup(gens[0].group, rels)
-        return FpGroup([])
+        return FpGroup(free_group([]), [])
```
Comment (jksuom) [inline sympy/combinatorics/fp_groups.py]:
```
>>> from sympy.combinatorics.free_groups import free_group
>>> from sympy.combinatorics.fp_groups import FpGroup
>>> FpGroup(free_group([]), [])
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "sympy/combinatorics/fp_groups.py", line 61, in __init__
    self.generators = self._generators()
  File "sympy/combinatorics/fp_groups.py", line 78, in _generators
    return self.free_group.generators
AttributeError: 'tuple' object has no attribute 'generators'
```

`free_group([])` returns a tuple (of length 1). `FreeGroup([])` should be used instead.

**Comment by fchapoton** [comment]
ok, done

**Comment by jksuom** [comment]
Thanks. It would be good to add a test as well. That reveals another error:
```
>>> from sympy.combinatorics.free_groups import FreeGroup
>>> from sympy.combinatorics.fp_groups import FpGroup, simplify_presentation
>>> simplify_presentation(FpGroup(FreeGroup([]), []))
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "sympy/combinatorics/fp_groups.py", line 933, in simplify_presentation
    change_gens=change_gens)
  File "sympy/combinatorics/fp_groups.py", line 939, in simplify_presentation
    identity = gens[0].group.identity
IndexError: tuple index out of range
```
In this example, `gens` is empty. It should be checked and handled as follows, for example:
```
         if gens:
             return FpGroup(gens[0].group, rels)
-        return FpGroup([])
+        return FpGroup(FreeGroup([]), [])
     elif len(args) == 2:
         gens, rels = args[0][:], args[1][:]
+        if not gens:
+            return gens, rels
         identity = gens[0].group.identity
     else:
         if len(args) == 0:
```
Can you add that? Then it would also be possible to add a test in sympy/combinatorics/tests/test_fp_groups.py, for instance:
```
def test_simplify_presentation():
    # ref #16083
    G = simplify_presentation(FpGroup(FreeGroup([]), []))
    assert not G.generators
    assert not G.relators
```

**Comment by jksuom** [comment]
Looks good. Only import simplify_presentation should be added to test_fp_groups.py.

**Pull request merged**